test/distributed/_tensor/test_convolution_ops.py

```diff
@@ -0,0 +1,181 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates
+# Owner(s): ["oncall: distributed"]
+
+import copy
+
+import torch
+import torch.nn as nn
+from torch.distributed._tensor import (
+    DeviceMesh,
+    distribute_module,
+    distribute_tensor,
+    Replicate,
+    Shard,
+)
+from torch.testing._internal.common_utils import run_tests
+from torch.testing._internal.distributed._tensor.common_dtensor import (
+    DTensorTestBase,
+    with_comms,
+)
+
+ITER_TIME = 10
+LR = 0.001
+
+
+def _conv_fn(
+    name: str,
+    module: nn.Module,
+    device_mesh: DeviceMesh,
+) -> None:
+    for name, param in module.named_parameters():
+        dist_spec = [Replicate()]
+        dist_param = torch.nn.Parameter(
+            distribute_tensor(param, device_mesh, dist_spec)
+        )
+        name = "_".join(name.split("."))
+        module.register_parameter(name, dist_param)
+
+
+class DistConvolutionOpsTest(DTensorTestBase):
+    @property
+    def world_size(self) -> int:
+        # hard code world size to 2
+        return 2
```

Comment:
shall we change the world size to 4 to align with other tests?

Reply:
Absolutely we can.

Reply:
Done.

Reply:
Thx for addressing the comment!